web_src/css/features/colorpicker.css

```diff
@@ -0,0 +1,159 @@
+/* This is a stripped-down version of coloris's CSS tailored to our needs. It does only include
+   opaqua colors, and if more features like opacity are needed, the CSS needs to be extended
+   based on upstream: https://github.com/mdbassit/Coloris/blob/main/src/coloris.css. */
```

Comment:
```suggestion
   based on upstream: https://github.com/melloware/coloris-npm/blob/main/src/coloris.css */
```

Reply:
Those repos have a bit of a strange relation, `mdbassit/Coloris` is the original source, `melloware/coloris-npm` is the npm distribution, I guess I prefer refering to it with the original file as it has the blame info and such.